minethd.cpp

```diff
@@ -33,7 +33,11 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
+#ifdef __MINGW64__
+	SetThreadAffinityMask(&h, 1ULL << cpu_id);
```

Comment:
Bottom line: MinGW accepts **&h** and not **h**, as far as my tests goes.

**You can hold, in fact you should hold this up until you're convinced!**